app/pages/lab-pages-editor/icons/CloseIcon.jsx

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable react/react-in-jsx-scope */
+
+export default function CloseIcon() {
+  return (
+    <span className="icon fa fa-close" role="img" />
```

Comment:
If this is supposed to be an image, it should allow for alt text to be passed in as an `aria-label` prop. All images must have alt text. If not, then remove the `role='img'`.